### PR TITLE
docs: README.md, CONTRIBUTING.md, doc.go

### PR DESCRIPTION
**Pull request by Anush008**
As the title goes.

### CODE REVIEW

File: tools/generate_proto_go.sh [hunk 1]
```diff
@@ -4,8 +4,8 @@ PROJECT_ROOT="$(pwd)/$(dirname "$0")/../"
 
 QDRANT_PROTO_DIR='proto'
 
-go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.34.2
-go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.5.1
+go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest
```
Comment (Robert-Stam) [inline tools/generate_proto_go.sh:8]: this is trick, are we sure we don't want to pin explicit versions for reproducibility?

**Review by Robert-Stam — COMMENTED** [review]
One question, further LGTM

File: tools/generate_proto_go.sh (hunk 1 above)
Comment (Anush008) [reply tools/generate_proto_go.sh:8]: Yeah. I had the doubt too.
I followed the instructions at https://grpc.io/docs/languages/go/quickstart/#prerequisites

Comment (Anush008) [reply tools/generate_proto_go.sh:8]: I think it's assuming backward compat, latest is recommended.

Comment (Anush008) [reply tools/generate_proto_go.sh:8]: Regardless, let me pin it to a version.

Comment (Anush008) [reply tools/generate_proto_go.sh]: [`9659554` (#54)](https://github.com/qdrant/go-client/pull/54/commits/9659554a250be3decdd11386779ec00bb18e6dd4)

Comment (Robert-Stam) [reply tools/generate_proto_go.sh]: 👍

**Pull request merged**